src/ReactNativeZoomableView.tsx

```diff
@@ -65,10 +69,17 @@ class ReactNativeZoomableView extends Component<
     contentHeight: undefined,
     panBoundaryPadding: 0,
     visualTouchFeedbackEnabled: true,
+    staticPinPosition: undefined,
+    staticPinIcon: undefined,
+    onStaticPinPositionChange: undefined,
+    onStaticPinPositionMove: undefined,
+    animatePin: true,
+    disableMomentum: false,
```

Comment:
while we're at it, can we also add an option to disable touch feedback?
https://github.com/openspacelabs/react-native-zoomable-view/issues/28

Reply:
Also https://github.com/openspacelabs/react-native-zoomable-view/issues/38 would be a great bonus if you're up for it